subsys/spm/Kconfig

```diff
@@ -9,6 +9,7 @@ menu "SPM"
 config SPM
 	bool "Use Secure Partition Manager"
 	default y if TRUSTED_EXECUTION_NONSECURE
+	select ARM_FIRMWARE_USES_SECURE_ENTRY_FUNCS
```

Comment:
I think `select` is bad practice. Use `imply` here instead, as you don't _HAVE TO_ use secure entry functions when using SPM.

Reply:
Done